### PR TITLE
[FIX] Disable non-NiftiMasker masks

### PR DESCRIPTION
**Pull request by tsalo**
References #466.

Changes proposed in this pull request:

- Check masker type in `nimare.utils.get_masker` and only allow `NiftiMasker`s.
- Remove IBMA test with custom masker.

### CODE REVIEW

File: nimare/tests/test_meta_ibma.py [hunk 1]
```diff
@@ -111,17 +107,6 @@ def test_ibma_smoke(testdata_ibma, meta, meta_kwargs, corrector, corrector_kwarg
         assert cres.get_map("z").ndim == 3
 
 
-def test_ibma_with_custom_masker(testdata_ibma):
```
Comment (tyarkoni) [inline nimare/tests/test_meta_ibma.py:114]: +1 for deleting this test, but maybe replace it with another small one that verifies an exception is raised if a different masker is passed?

**Review by tyarkoni — APPROVED** [review]
LGTM, just one minor comment you can ignore if you like.

**Comment by tyarkoni** [comment]
LGTM!

**Comment by tsalo** [comment]
Closing since we're going to pivot to blocking NiftiLabelsMaskers for some estimators and just warning for others.

**Pull request closed without merging**